doc/03_research/Leaderboard-2/changes_leaderboard2.md

```diff
@@ -0,0 +1,90 @@
+# Overview leaderboard 2.0
+
+**Summary:** New Features and changes made with the CARLA leaderboard-2.0
+
+---
+
+## Author
+
+Samuel Kühnel
+
+## Date
+
+17.11.2023
+
+## General Information
+
+Leaderboard 1.0             |  Leaderboard 2.0
+:-------------------------:|:-------------------------:
+![leaderboard-1](../../00_assets/leaderboard-1.png)  |  ![leaderboard-2](../../00_assets/leaderboard-2.png)
+
+As shown in the images above the new leaderboard seems to have way more traffic than the previous version. The leaderboard 2.0 uses an enhanced version of CARLA 0.9.14. So be aware that even if the documentation mentions this version tag, there are probably features missing.
+Therefore it is recommended to use the latest version.
+
+### Submissions
+
+Every team can submit five times per month to the CARLA Leaderboard 2.0. The number of submissions is reset every month automatically. The full start guide and documentation can be found [here](https://leaderboard.carla.org/get_started/).
+
+## New Features
+
+### Maps
+
+In the Leaderboard 2.0 new and larger maps were added, [town 12](https://carla.readthedocs.io/en/latest/map_town12/) and [town 13](https://carla.readthedocs.io/en/latest/map_town13/).
+These maps consist of different regions e.g. downtown zone with high bulidngs arranged in blocks or rural areas. The city is surrounded by a highway system with 3-4 lanes and intersections/junctions.
+
+The evaluation of submissions will be done on a secret map **town 14** that is kept secret but is designed similar to town 12 and 13.
+
+### Szenarios and training database
+
+The new leaderboard comes with a large set of (new) szenarios that the autonomous agent has to accomplish. Some examples are merging lanes, emergency vehilces or Accidents. For each of the new scenrios CARLA provides logs from a $100 \%$ finish.
```

Comment:
- "vehicles"
- "accidents"
- "scenarios" (2x)
- The percent sign is not rendering

Just some really small stuff :)